src/website/design/content-redesign-notes.md

```diff
@@ -0,0 +1,92 @@
+# Website Layout
+
+This document provides an overview of the content design for the revised openapis.org website.
+
+## Menu
+
+This section provides the content of the menu bar.
+
+### Drop downs
+
+The following are the drop downs we should aim for.
+
+```text
+Learn
+├── What Is OpenAPI? (Relocate existing page).
+├── [Learn About OAI](https://learn.openapis.org/)
+├── [OAI Courses](https://training.linuxfoundation.org/full-catalog/?_sf_s=openapi)
+Specifications
+|
+├── [OpenAPI](https://spec.openapis.org/oas)
+├── [Arazzo](https://spec.openapis.org/arazzo)
+├── [Overlay](https://spec.openapis.org/overlay)
+├── [Registries](https://spec.openapis.org/registry/)
+Community
+├── [Collaborate on Slack](https://communityinviter.com/apps/open-api/openapi)
```

Comment:
We're going to have Slack/GitHub icons. Do we need a dedicated menu item for Slack? Though, i think it might be important to have a page to describe how collaboration happens in OAI. E.g. the main communication channel is Slack, it's divided into sub-channels for each spec/SIG, etc. Also, how to join community meetings, not everyone knows where to find this information.